modules/security/pages/gsql-file-input-policy.adoc

```diff
@@ -0,0 +1,430 @@
+= File Input Policy
+
+GSQL file input policy allows users to apply restrictions on the location of local files used to load data to TigerGraph.
+
+== Allowlist and Blocklist
+
+The policy consists of an _allowlist_ and a _blocklist_:
+
+* Only paths that belongs to one of the `allowlists` will be allowed for input.
+* The blocklist has higher priority over the `allowlist`.
+** Ex. If a path belongs to both the `allowlist` and `blocklist` simultaneously, the path will be blocked.
+* Users can use `gadmin config` to set the `blocklist` and `allowlist` for the file input policy.
+
+[console]
+----
+gadmin config set GSQL.FileInputPolicy '["/__TG_DATA_ROOT__/gui/loading_data", "!/etc", "!/usr/lib"]'
+gadmin config apply -y
+gadmin restart gsql restpp -y
+----
+
+Similar to the xref:tigergraph-server:security:file-output-policy.adoc[] the value for the config is a list of strings that start with either `!/` for the `blocklist` or `/` for the `allowlist`.
+
+== Format
+
+The format of an entry in the `blocklist` or `allowlist` follow the rules below:
+
+* Paths can use data root: `__TG_DATA_ROOT__`, which will be translated to actual data root when it’s used.
++
+[NOTE]
+This is the same as `gadmin config get System.DataRoot`.
+
+* Paths must be an absolute path.
+** Paths must start with either `/` for `allowlist`, or `!/` for `blocklist`.
+** Dots are *not* allowed in the path (EX. `/a/../b`, `/a/./b`).
+** Paths should not end with `/` (EX.`/tmp/ → /tmp`).
+*** The only exception is for the root path.
+** Some commands will convert the input relative path to an absolute path with the GSQL client working directory (where the GSQL client was invoked)
```

Comment:
may better add "implicitly" here: `will implicitly convert`

Reply:
Good point. I added this with last commit.